### PR TITLE
Update go directive value in go.mod files to 1.19

### LINKED ISSUE #3839: Update go directive value in go.mod files to 1.19
Shouldn't we bump Go to 1.19 in `go.mod` files per https://github.com/open-telemetry/opentelemetry-go/pull/3813?

### PR DESCRIPTION
**Pull request by EvanMu96**
Fix #3839

### CODE REVIEW

**Comment by pellared** [comment]
@EvanMu96 Can you sign the CLA? More info: https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md

**Comment by EvanMu96** [comment]
> @EvanMu96 Can you please sign the CLA? More info: https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md

ok, thanks!

File: bridge/opencensus/metric.go [hunk 1]
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build go1.18
-// +build go1.18
+//go:build go1.19
+// +build go1.19
```
[the remark this answers is not in the document]
Comment (EvanMu96) [reply bridge/opencensus/metric.go:16]: > Is this needed? Why?
sorry this one is not needed. I will fix it.

[the remark this answers is not in the document]
Comment (pellared) [reply bridge/opencensus/metric.go]: Can we this entirely?

File: bridge/opencensus/metric.go [hunk 2]
```diff
@@ -12,9 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build go1.18
-// +build go1.18
-
```
Comment (pellared) [inline bridge/opencensus/metric.go:17]: @dashpole Do you happen to know why these build tags were here?

**Review by pellared — APPROVED** [review]

**Review by MadVikingGod — APPROVED** [review]

**Review by hanyuancheung — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Pull request merged**